sdk/src/wallet/core/operations/background_syncing.rs

```diff
@@ -49,16 +49,16 @@ where
         tx_background_sync.send(BackgroundSyncStatus::Running).ok();
 
         let wallet = self.clone();
+        let interval_seconds = interval.unwrap_or(DEFAULT_BACKGROUNDSYNCING_INTERVAL);
 
         task::spawn(async move {
             loop {
                 log::debug!("[background_syncing]: syncing wallet");
 
-                if let Err(err) = wallet.sync(options.clone()).await {
+                if let Err(err) = wallet.sync(interval_seconds.clone()).await {
```

Comment:
was this on purpose? shouldn't it still be `options`?

Reply:
Man... my vscodes crashed alongside my terminal (I blame rust-analyzer) and when I opened it again I went too fast on editing... :rofl: Anyway, just fixed it in the last commit  😅